### DISCUSSION TITLE
Mapping the back paddles of the 8BitDo Ultimate Bluetooth Controller

### DISCUSSION THREAD

**Discussion by andrew-wright** [AntiMicroX/antimicrox#610] [Q&A]
Hello. I recently bought 8BitDo's Ultimate Bluetooth Controller, and on the back of the controller are two paddles, and I thought it would be handy to map those (either one or both) to the play/pause media command in Windows. The reason why I'd like to do this is so that I can play/pause my music/videos (something I have to do often) using the paddles. I setup my mouse's thumb button to do exactly that, and I find it extremely convenient, and use it several dozen times per day. So I thought it would be a great use for the back paddles, which I currently have mapped to nothing.

Is it possible to do that using AntiMicroX? I opened the app and toyed around with it, and I see that it's recognizing all of the 8BitDo controller's button presses, _other_ than the two back paddles. Is there something I can do to make those be recognized, so that I can remap them? (And for context, the controller does come with 8BitDo's "Ultimate Software," which allows the back paddles to be remapped, but they can only be remapped to use other button inputs already on the controller, so I'm not able to remap it to use a keyboard keypress.)

Any ideas if what I'm hoping is possible? Thanks!

**Comment by pktiuk** [comment]
Are paddles in this gamepad recognized bu another apps? (Games or apps like jstest-gtk, qjoypad or [sdl-jstest](https://github.com/Grumbel/sdl-jstest))
